src/app/views/common/share.ts

```diff
@@ -27,23 +27,23 @@ export const createShareLink = (sampleQuery: IQuery, authenticated?: boolean): s
 
   if (sampleBody && Object.keys(sampleBody).length > 0) {
     const requestBody = hashEncode(JSON.stringify(sampleBody));
-    shareLink = `&requestBody=${requestBody}`;
+    shareLink += `&requestBody=${requestBody}`;
   }
 
   if (sampleHeaders && sampleHeaders.length > 0) {
     const headers = hashEncode(JSON.stringify(sampleHeaders));
-    shareLink = `${shareLink}&headers=${headers}`;
+    shareLink += `${shareLink}&headers=${headers}`;
```

Comment:
shouldn't this be removed or stay as before?

Reply:
It should actually stay as before